hyperwallet/api.py

```diff
@@ -114,7 +116,7 @@ def getUser(self,
             raise HyperwalletException('userToken is required')
 
         response = self.apiClient.doGet(
-            os.path.join('users', userToken)
+            build_url('users', userToken)
```

Comment:
build_url method name must follow the naming convention of surrounding code, so it should be like buildUrl()

Reply:
ok, just wanted to name the method according to the Python Style Guide https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables